ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -2408,7 +2408,7 @@ and perform_exn ?subtask ?result (op : operation) (t : Xenops_task.task_handle)
         in
         debug "%s compress memory: %b" __FUNCTION__ compress_memory ;
         let verify_cert =
-          if vmm.vmm_verify_cert then Stunnel_client.pool () else None
+          if vmm.vmm_verify_cert then Some Stunnel.pool else None
```

Comment:
This is confusing, but it's exactly what the doctor ordered

Reply:
I agree...

Reply:
maybe a comment explaining the situation would help?